### PR TITLE
chore(node): don't store client peer sessions

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

**Comment by joshuef** [comment]
bors try

**Comment by joshuef** [comment]
bors try

**Review by RolandSherwin — APPROVED** [review]

File: sn_node/src/comm/mod.rs [hunk 1]
```diff
@@ -113,6 +113,10 @@ impl Comm {
 
         let sessions = self.sessions.clone();
 
+        // TODO: Is there an optimium we should actually have.
+        // Assuming we dont store clients... can we test for this
+        trace!("Sessions known of: {:?}", sessions.len());
```
Comment (oetyng) [inline sn_node/src/comm/mod.rs:118]: Hmm.. should correlate with how many members we have in the section roughly? 
And then some for contacts with other sections..

So, we could compare sessions count with our network knowledge basically.

There should be at most
- one per node in our section
- one per Elder in other sections we know of

I think?

**Pull request merged**